src/theme/index.ts

```diff
@@ -192,6 +192,24 @@ const Theme = {
     },
   },
 
+  scrollbar: {
+    scrollbarGutter: 'stable both-edges',
+    'overflow-y': 'hidden',
+    ':hover': {
+      'overflow-y': 'auto',
+    },
+    '::-webkit-scrollbar': {
+      width: 8,
+      background: 'var(--white-900, rgba(255, 255, 255, 0.10))',
```

Comment:
I don't think we have CSS vars defined anywhere with --white-900. it should be removed
```suggestion
      background: 'var(rgba(255, 255, 255, 0.1))',
```